core/api/service/api_service.cpp

```diff
@@ -36,16 +57,54 @@ namespace kagome::api {
     for (const auto &listener : listeners_) {
       auto on_new_session =
           [wp = weak_from_this()](const sptr<Session> &session) mutable {
-            if (auto self = wp.lock(); not self) {
+            auto self = wp.lock();
+            if (!self) {
               return;
             }
+
+            if (SessionType::kSessionType_Ws == session->type()) {
+              auto subscribed_session =
+                  self->store_session_with_id(session->id(), session);
+              subscribed_session->set_callback([wp](SessionPtr &session,
+                                                    const auto &key,
+                                                    const auto &data,
+                                                    const auto &block) {
+                if (auto self = wp.lock()) {
+                  jsonrpc::Value::Array out_data;
+                  out_data.emplace_back(api::makeValue(key));
+                  out_data.emplace_back(api::makeValue(data));
+
+                  /// TODO(iceseer): make event notofication depending in
```

Comment:
Add task id, please.

Reply:
There is no task. And referring to other ```TODO```s in the code it's unnecessary.